src/Simplex/Messaging/Agent/Store/SQLite/Migrations/M20240417_rcv_files_only_via_proxy.hs

```diff
@@ -0,0 +1,18 @@
+{-# LANGUAGE QuasiQuotes #-}
+
+module Simplex.Messaging.Agent.Store.SQLite.Migrations.M20240417_rcv_files_only_via_proxy where
+
+import Database.SQLite.Simple (Query)
+import Database.SQLite.Simple.QQ (sql)
+
+m20240417_rcv_files_only_via_proxy :: Query
+m20240417_rcv_files_only_via_proxy =
+    [sql|
+ALTER TABLE rcv_files ADD COLUMN only_via_proxy INTEGER NOT NULL DEFAULT 0;
```

Comment:
```suggestion
ALTER TABLE rcv_files ADD COLUMN approved_relays INTEGER NOT NULL DEFAULT 0;
```

Reply:
so the agent wouldn't need to know whether the relays were approved by the user or because they were in the list of known relays at the time download started.